docs/src/code-examples/generate-template-preview/go.mdx

```diff
@@ -3,22 +3,27 @@ lang: go
 ---
 
 ```go
-resp, err := client.GenerateTemplatePreview(id, 
+resp, err := client.GenerateTemplatePreview(id,
     notify.Personalisation{
-        {"name", "John"},
+        {"name", "Kim"},
         {"day", "Monday"},
+        {"colours", []string{"red", "blue"}},
     },
 )
 ```
+
 The parameters in the personalisation argument must match the placeholder fields in the actual template.
 
 <Panel label={<H3 appearAs={H5}>Arguments</H3>}>
   <Parameters>
     <Parameter required name="id" kind="string">
-      Sign in to Notify and go to the Templates page to find the template
-      ID.
+      Sign in to Notify and go to the Templates page to find the template ID.
     </Parameter>
-    <Parameter optional name="personalisation" kind="[]struct{ Key, Value string }">
+    <Parameter
+      optional
+      name="personalisation"
+      kind="[]struct{ Key string, Value interface{} }"
```

Comment:
Should we update the examples for the other languages as well?

Reply:
Yep, good call, I forgot.

Reply:
Going to update the other languages separately, possibly one at a time.

Reply:
Updated node, will do others separately.